README.md

```diff
@@ -60,7 +60,19 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct. This tables describes shows overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
+
+|                   |LVM                    |Direct              |
+|:--                |:--                    |:--                 |
+|Purpose            |avoids fragmentation   |simpler, somewhat faster, but storage may get fragmented   |
+|What is served     |LVM Logical Volume     |pmem block device   |
+|Region-affinity    |yes:One Volume Group per Region  |yes: namespace can belong to one Region only  |
+|Startup            |Two extra stages: pmem-ns-init(creates namespaces), vgm(creates Volume Groups)   |No extra steps at startup |
+|Namespace modes    |fsdax, sector -mode namespaces precreated as pools   |Namespace of required mode created directly, no need to pre-create pools   |
```

Comment:
Why the hyphen before "mode"?

Reply:
"precreated" -> "pre-created"

Reply:
hyphen is because I was thinking "fsdax-mode and sector-mode namespaces", then used comma to combine those, and then single hyphen remained. I will remove hyphen, lets see how it looks